src/main/java/com/brandon3055/draconicevolution/datagen/LangGenerator.java

```diff
@@ -726,6 +728,8 @@ private void misc(PrefixHelper helper) {
         add("key.draconicevolution.dislocator_gui",                             "Advanced Dislocator GUI");
         add("key.draconicevolution.dislocator_up",                              "Advanced Dislocator Select Up");
         add("key.draconicevolution.dislocator_down",                            "Advanced Dislocator Select Down");
+
+        add("info.de.hasSavedData.txt",                                         "Contents saved!");
```

Comment:
That does not belong there. Just ignore that localization for now. I need to update it.